File: csharp/TwoSum.cs
public class Solution {
    public int[] TwoSum(int[] nums, int target) {
        for (int i = 0; i < nums.Length; i++){
            for (int y = i + 1; y < nums.Length; y++){
                if (nums[y] + nums[i] == target)
                    return new int[] {i, y};
            }
        }
        
        return new int[] {0, 0};
    }
}

File: cpp/twosum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        for (int x = 0; x != nums.size(); ++x){
            for (int y = x + 1; y != nums.size(); ++y){
                if (nums[x] + nums[y] == target){
                    return vector<int> { x, y };
                }
            }
        }
        
        return vector<int> { 0, 0 };
    }
};